readme.md

```diff
@@ -13,14 +13,30 @@
 
 ## Usage
 
-Import the desired theme into Alacritty's config:
+1. Locate (or create) Alacritty's config file (refer to [the *Configuration* section of Alacritty's readme](https://github.com/alacritty/alacritty/tree/master#configuration)).
```

Comment:
Because the other points don't end in a period I think we can remove this one for consistency

```suggestion
1. Locate (or create) Alacritty's config file (refer to [the *Configuration* section of Alacritty's readme](https://github.com/alacritty/alacritty/tree/master#configuration))
```